Release 2.25.2 (#9348)
timescale/timescaledb: CHANGELOG.md

you use psql to update your database, use the -X flag and prevent any .psqlrc 
commands from accidentally triggering the load of a previous DB version.**

## 2.25.2 (2026-03-03)

> This release contains performance improvements and bug fixes since the 2.25.1 release and a fix for a security vulnerability (#9331). We recommend that you upgrade at the next available opportunity.

Reviewer: ```suggestion
This release contains performance improvements and bug fixes since the 2.25.1 release and a fix for a security vulnerability (#9331). You can check the security advisory(https://github.com/timescale/timescaledb/security/advisories/) 
for more information on the vulnerability and the platforms that are affected.
We recommend that you upgrade as soon as possible.
```

for more information on the vulnerability and the platforms that are affected.

> We recommend that you upgrade as soon as possible.

Reviewer: ```suggestion
This release contains performance improvements and bug fixes since the 2.25.1 release and a fix for a security vulnerability (#9331). You can check the security advisory(https://github.com/timescale/timescaledb/security/advisories/GHSA-vgp2-jj5c-828m) 
for more information on the vulnerability and the platforms that are affected. We recommend that you upgrade as soon as possible.
```

you use psql to update your database, use the -X flag and prevent any .psqlrc 
commands from accidentally triggering the load of a previous DB version.**

## 2.25.2 (2026-03-03)

This release contains performance improvements and bug fixes since the 2.25.1 release and a fix for a security vulnerability (#9331). We recommend that you upgrade at the next available opportunity.

**Highlighted features in TimescaleDB v2.25.2**
* 

**Backward-Incompatible Changes**

**Features**
* Overlap check for concurrent continuous aggregate refresh

> 

Reviewer: ```suggestion
```
